NDTensors/src/diag/tensoralgebra/contract.jl

```diff
@@ -58,6 +58,26 @@ function _contract!!(
   return R
 end
 
+function contract!(
+  output_tensor::Exposed{<:AbstractArray,<:DiagTensor},
+  labelsoutput_tensor,
+  tensor1::Exposed,
+  labelstensor1,
+  tensor2::Exposed,
+  labelstensor2,
+  α::Number=one(Bool),
+  β::Number=zero(Bool),
+)
+  return contract!(
```

Comment:
I found that I needed this function because the `contract(::DiagTensor, any, ::DiagTensor, any, ::DiagTensor, any)` function below does not have `alpha` and `beta`.

Reply:
check here that \alpha and \beta are [1, 0]

Reply:
I added asserts to verify that alpha was approx 1.0 and beta was approx 0.0